tools/rails.md

```diff
@@ -0,0 +1,33 @@
+---
+permalink: /rails
+layout: post
+title: Rails
+link: https://guides.rubyonrails.org/maintenance_policy.html
+releaseDateColumn: true
+category: framework
+sortReleasesBy: release
+releases:
+  - releaseCycle: 5.2
+    release: 2018-04-09
+    eol: false
+    latest: 5.2.3
+  - releaseCycle: 5.1
+    release: 2017-04-27
+    eol: false
+    support: 2018-04-09
+    latest: 5.1.7
+  - releaseCycle: 5.0
+    release: 2016-06-30
+    eol: 2018-04-09
+    support: 2018-04-09
+    latest: 5.0.7.2
+  - releaseCycle: 4.2
+    release: 2014-12-20
+    eol: 2017-04-27
+    support: 2016-06-30
+    latest: 4.2.11.1
+---
+
+Only the latest Rails version gets bug fixes, and the version before that gets security fixes. Severe security issues (as judged by the core team) are backported further; e.g., v4.2.11.1 and v5.0.7.2 are [severe security fixes](https://weblog.rubyonrails.org/2019/3/13/Rails-4-2-5-1-5-1-6-2-have-been-released/) that were created after v4.2 and v5.0 were no longer receiving (non-severe) security updates.
+
+A complete list of historic versions is available on [RubyGems](https://rubygems.org/gems/rails/versions). New releases are published on the [Rails blog](https://weblog.rubyonrails.org/releases/).
```

Comment:
Maybe it's worth mentioning that there is commercial long term support for older Rails versions as well: <https://railslts.com/#how-it-works>

Reply:
I don't want to link to a commercial product I have no experience with, but if you want you can send in another PR after this one gets merged.